platformio.ini

```diff
@@ -280,6 +283,8 @@ board_build.embed_files = assets/bmp/brokenclouds.jpg
                           assets/bmp/snow.jpg
                           assets/bmp/snownight.jpg
                           assets/bmp/thunderstorm.jpg
+                          assets/gif/colorsphere.gif
+                          assets/gif/atomic.gif
```

Comment:
Maybe it's best to put these two lines directly underneath assets/bmp/thunderstormnight.jpg, so that the weather effect images stay together.